components/home/index.js

```diff
@@ -34,27 +36,37 @@ class Home extends Component {
         longitudeDelta: 0.05
       }
     });
-  }
+  };
 
-  // Function : change the visiblity of the switchCampuses component
-  // parameter: boolean to set the visibility (false: unvisible)
-  changeVisibilityTo = (visibility) => {
-    this.setState({ isVisible: visibility });
+  changeVisibilityTo = boolean => {
+    this.setState({ isVisible: boolean });
+  };
+  changeVisibilityToSearch = boolean => {
+    this.setState({ isVisible2: boolean})
   }
 
-  // Function: Updates coordinates state to draw polyline
-  // Parameter: object with latitudes and longitudes
-  updateCoordinates = (newCoordinates) => {
+  updateRegion2 = newRegion2 => {
```

Comment:
Why do you have a second function with the same name? Is this necessary

Reply:
this function is used to update the region in the second search bar.